book/website/project-design/missing-data/missing-data-structures.md

```diff
@@ -0,0 +1,129 @@
+
+> **Example**: Blood pressure readings may be missing from individuals who are older, frailer, and have less mobility, and therefore, are more likely to not attend the clinic. In this instance, the reason data is missing in the variable of interest (blood pressure), is related to other observed variables (age and mobility). 
+>
+> | Participant Number | Age | Diastolic Blood Pressure | Systolic Blood Pressure | Blood Test Result                                  | Motor Score | Cognitive Score |
+> |--------------------|-----|--------------------------|-------------------------|---------------------------------------------------|-------------|-----------------|
+> | 1                  | 56  | 82                       | 118                     | <span style="color:red;"><strong>N/A</strong></span> | 10          | 35              |
+> | 2                  | 78  | 87                       | 134                     | <span style="color:red;"><strong>N/A</strong></span> | 32          | 29              |
+> | 3                  | 85  | <span style="color:red;"><strong>N/A</strong></span> | <span style="color:red;"><strong>N/A</strong></span> | <span style="color:red;"><strong>N/A</strong></span> | 27          | 14              |
+> | 4                  | 43  | 83                       | 121                     | Negative                                           | 15          | 36              |
+> | 5                  | 67  | 86                       | 131                     | Positive                                           | 20          | 25              |
+> | 6                  | 82  | 92                       | 133                     | Negative                                           | 26          | 12              |
+> | 7                  | 88  | <span style="color:red;"><strong>N/A</strong></span> | <span style="color:red;"><strong>N/A</strong></span> | Positive                                           | 34          | 10              |
+> | 8                  | 71  | <span style="color:red;"><strong>N/A</strong></span> | <span style="color:red;"><strong>N/A</strong></span> | Negative                                           | 33          | 22              |
+>
+> Individuals with a high motor score (>26) and who were older (>70) were unable to attend the blood pressure clinic. 
+
+
+
+(pd-missing-data-structures-mnar)=
+## Missing not at Random (MNAR)
+
+Data that are MNAR is missing due to reasons that we do not know. In other words, the reason for the missingness is related to the value of the variable that is missing. This is the most complex case of data missingness to handle, as bias may occur but cannot be adjusted for as the source of the missingness is unmeasured. 
```

Comment:
Data that are MNAR is missing due to reasons that we do not know.

maybe it should be 'data that are MNAR are missing' instead of is? but again I might be wrong on this one!